components/brave_perf_predictor/browser/perf_predictor_tab_helper_browsertest.cc

```diff
@@ -138,6 +140,7 @@ IN_PROC_BROWSER_TEST_F(PerfPredictorTabHelperTest, NewNavigationStoresSavings) {
   GURL second_url =
       embedded_test_server()->GetURL("example.com", "/blocking.html");
   ui_test_utils::NavigateToURL(browser(), second_url);
+  ASSERT_TRUE(ExecuteScript(contents, "addImage('logo.png')"));
```

Comment:
mind replacing all `ExecuteScript` with `EvalJs/ExecJs` (which is a recommended API now)?